src/shogun/base/Parallel.cpp

```diff
@@ -43,14 +54,14 @@ Parallel::~Parallel()
 int32_t Parallel::get_num_cpus() const
 {
 #if defined(LINUX) && defined(_SC_NPROCESSORS_ONLN)
-		return sysconf( _SC_NPROCESSORS_ONLN );
```

Comment:
usually, we try to avoid whitespace changes in such patches, and would send a separate patch